### ISSUE TITLE
Flickering when clicking any button. 

### ISSUE THREAD

**Issue by andrew749** [pahgawk/Pluto#12]
I guess it reloads the page whenever we click a button, rather than just making a javascript request.

**Comment by davepagurek** [comment]
Right, my sketchy way of dealing with our lack of client-side js (which I want to continue to avoid for the most part) is to reload the page, but make it scroll to the same spot with the anchor in the url. Any ideas for a better way?
